phpcs.xml.dist

```diff
@@ -22,11 +22,17 @@
         <!-- Exclude sniffs that require newer PHP versions -->
         <!-- ********************************************** -->
 
+        <!-- Requires PHP 7.3 -->
+        <exclude name="SlevomatCodingStandard.Functions.RequireTrailingCommaInCall.MissingTrailingComma" />
```

Comment:
Out of curiosity, how does doctrine/coding-standard utilize these sniffs if it claims compatibility with PHP 7.2+ (see: [composer.json](https://github.com/doctrine/coding-standard/blob/11.1.0/composer.json#L25)). Does this have something to do with the version-specific patch files?

Reply:
I'd have to run some tests to figure this one out, but it is possible that specifying a PHP version in the phpcs configuration (like `<config name="php_version" value="70200"/>`) would fix this. IIRC it depends on the sniff actively checking for it though, so it may not work for every sniff. That said, I'm fine excluding such sniffs manually until we upgrade to the required PHP version.

Reply:
Ah, that makes sense. I do see a PHP version check in [RequireTrailingCommaInCallSniff](https://github.com/slevomat/coding-standard/blob/8.10.0/SlevomatCodingStandard/Sniffs/Functions/RequireTrailingCommaInCallSniff.php#L47).

Per our earlier Slack conversation where you said: 

> I can only imagine it’s an issue due to us running phpcs on PHP 7.4 (e.g. an older version of phpdoc-parser that doesn’t support object shapes). You can try adding `<config name="php_version" value="70400"/>` to phpcs.xml.dist and then running phpcs on a newer PHP version to see if that fixes the issue

I misunderstood this and thought we were running `phpcs` with syntax rules for 7.4, which isn't the case. I now see you were just referring to the GitHub Action running `phpcs` through PHP 7.4.

I'll play around with this PR some more. I think adding `<config name="php_version" value="70200"/>` (and keeping that in sync with our minimum supported version) would be best if that allows us to remove the "Exclude sniffs that require newer PHP versions" block here.

Reply:
Yes, I believe I confused this with psalm where the `phpVersion` config does something else. Adding the `php_version` configuration and keeping it in sync with our minimum required version sounds sensible to me.